settings.py

```diff
@@ -26,6 +26,12 @@
 
 # APP AND MIDDLEWARE SETTINGS
 
+SOCIAL_AUTH_AUTHENTICATION_BACKENDS = (
+    'social_core.backends.spotify.SpotifyOAuth2',
+)
```

Comment:
Can you also add `'django.contrib.auth.backends.ModelBackend',` to this list? That way we can still have an administrative user that can login via username and password credentials?